pkg/operator/starter.go

```diff
@@ -139,12 +140,31 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	certRotationBase := time.Duration(0)
+	err = wait.PollImmediate(time.Second, 1*time.Minute, func() (bool, error) {
```

Comment:
has this leaked other places?  smells like a library function if we're going to do it this way.

Reply:
Yes, we have identical code in kas-o's starter.go